tests/scripts/thread-cert/run_cert_suite.py

```diff
@@ -39,7 +39,9 @@
 
 THREAD_VERSION = os.getenv('THREAD_VERSION')
 VIRTUAL_TIME = int(os.getenv('VIRTUAL_TIME', '1'))
-MAX_JOBS = multiprocessing.cpu_count() * 2 if VIRTUAL_TIME else 10
+_DEFAULT_MAX_JOBS = multiprocessing.cpu_count() * 2 if VIRTUAL_TIME else 10
```

Comment:
Not every familiar with python, does `multiprocessing.cpu_count() * 2 if VIRTUAL_TIME else 10` evaluate to `(multiprocessing.cpu_count() * 2) if VIRTUAL_TIME else 10` or `multiprocessing.cpu_count() * (2 if VIRTUAL_TIME else 10)`?

Reply:
Good question. I think it's the former.